examples/api/custom_projection_example.py

```diff
@@ -291,12 +291,12 @@ class DegreeFormatter(Formatter):
         """
 
         def __init__(self, round_to=1.0):
+            super(HammerAxes.DegreeFormatter, self).__init__()
             self._round_to = round_to
 
-        def __call__(self, x, pos=None):
+        def format_for_tick(self, x, pos=None):
```

Comment:
The burden is the same. I am just trying to say that if documentation is omitted - which might be the case for a quickly written and very simple derived class - the name of the method cannot help you to understand what it does.